packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -305,6 +313,7 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     },
   });
 
+  // TODO do we need to wrap this in `if (!baggage.has(firstIncarnationKey))` ?
```

Comment:
depends on whether `settlerKit` persists. it was designed at first to be per incarnation